noir-projects/noir-contracts/contracts/claim_contract/src/main.nr

```diff
@@ -28,7 +28,7 @@ pub contract Claim {
     }
 
     #[private]
-    fn claim(proof_note: UintNote, recipient: AztecAddress) {
+    fn claim(proof_note: UintNote, note_storage_slot: Field, recipient: AztecAddress) {
```

Comment:
It's strange to see a storage_slot being passed to a top-level function. I don't understand what this contract does, but can this storage slot not be derived from other information within the body of the function? What's a proof_note?

Reply:
You are right. And it actually turned out to be quite straightforward to address: https://github.com/AztecProtocol/aztec-packages/pull/11942/commits/6f807b8a28b3abf1bebd6f07e9052b351dc40f81

Reply:
It seems like this hasn't been addressed? The commit is not even on the repo